src/main/java/org/opensearch/geospatial/index/mapper/xypoint/ParsedXYPoint.java

```diff
@@ -23,11 +23,19 @@ public void normalize(String fieldName) {
         // normalization is not required for xy_point
     }
 
+    /**
+     * @param x x coordinate
+     * @param y y coordinate
+     * to set x and y values of XYPoint
```

Comment:
```suggestion
    /**
     * To set x and y values of XYPoint
     * @param x x coordinate
     * @param y y coordinate
     * 
```

Fix the documentation at all the places. Also, please see how other functions are documented in the repo.

Reply:
Sure, will update it